packages/eslint-plugin-mdx/src/rules/remark.ts

```diff
@@ -11,13 +11,18 @@ import vfile from 'vfile'
 import { getPhysicalFilename, getRemarkProcessor } from './helpers'
 import type { RemarkLintMessage } from './types'
 
-const processSync = createSyncFn(require.resolve('../worker')) as (
-  fileOptions: VFileOptions,
-  physicalFilename: string,
-  isFile: boolean,
-) => {
-  messages: VFile['messages']
-  content: string
+// call `creatSyncFn` lazily for performance, it is already cached inside, related #323
+const _ = {
```

Comment:
It may be useful to give this a more specific name.
Maybe something like
```suggestion
const lazyLoadMdxProcessor = {
```

Reply:
Too long to use `lazyLoadMdxProcessor.processSync` everywhere to me. Ideally it should be a simple `processSync`, but impossible in js maybe. The comment above already shows what we're doing here.

Reply:
A shorter alternative would be fine as well.
I think there's something between `_`, which tells developers nothing about what this is or what it does, and `lazyLoadMdxProcessor`, which you consider too verbose. :slightly_smiling_face:

Reply:
Yeah, I was thinking for a shorter and better alternative too. But my English is not so good to have enough short word to describe it, any advice?

Reply:
Is a short `lazy` good enough? `lazy.processSync` makes sense to me.

Reply:
Maybe `lazyRemark`? or `lazyMdx`?
`lazy` could also gives more info, but could be ambiguous as to what is being loaded.

Reply:
or if there isn't another processor in this scope, and it's a wrapper around the processor maybe plain `remark`, and the laziness could be noted in a comment (which you've already added)

Reply:
`remark` is already a const variable as the rule name, so I choose `lazyRemark` finally.